README.md

```diff
@@ -992,22 +983,22 @@ Converts a given integer or base 10 string representing an integer to a specifie
   * `convert_base(5, 2)` results in: '101'
   * `convert_base('254', '16')` results in: 'fe'
 
-Note: Since Puppet 4.5.0 this can be done with the built-in [`String.new`](https://puppet.com/docs/puppet/latest/function.html#integer-to-string) function and its many formatting options:
+Since Puppet 4.5.0, this can be done with the built-in [`String.new`](https://puppet.com/docs/puppet/latest/function.html#integer-to-string) function, with various formatting options:
 
     $binary_repr = String(5, '%b') # results in "101"
     $hex_repr = String(254, '%x')  # results in "fe"
     $hex_repr = String(254, '%#x') # results in "0xfe"
 
 #### `count`
 
-Takes an array as first argument and an optional second argument.
-Counts the number of elements in array that is equal to the second argument.
-If called with only an array it counts the number of elements that are not nil/undef/empty-string.
+Takes an array as the first argument and an optional as the second argument.
+It counts the number of elements in an array that is equal to the second argument.
+If called with only an array, it counts the number of elements that are not nil/undef/empty-string.
 
-**Note**: Equality is tested with a Ruby method and it is therefore subject to what Ruby considers
-to be equal. For strings this means that equality is case sensitive.
+> **Note**: Equality is tested with a Ruby method. It is subject to what Ruby considers
+to be equal. For strings, equality is case sensitive.
```

Comment:
The phrase changed here is in many places. Fine to change here, but should perhaps be modified everywhere (in files not touched by this set of updates to README).